build/style_test.go

```diff
@@ -706,7 +687,10 @@ func TestStyle(t *testing.T) {
 			t.Fatal(err)
 		}
 
-		if err := stream.ForEach(filter, func(s string) {
+		if err := stream.ForEach(stream.Sequence(
+			filter,
+			stream.GrepNot(`: (field no|type No)Copy is unused \(U1000\)$`),
```

Comment:
Ah. Please file an issue (in `unused`) if you want this strongly. It'll at least put it on my todo list and I'll be able to think about an interface for it.